### DISCUSSION TITLE
ETCTV.cl channel shows up in the channels database, but it is not present in the m3u file?

### DISCUSSION THREAD

**Discussion by kratoz29** [iptv-org/community#558] [Q&A] [answered]
Sorry if this is a noob question, but I was looking for said channel and I found it in the csv database, therefore I thought it was available through the CL country or Animation group, but it is not showing up, I also did not see it in the banned channels list.

Is the channel list unrelated to what streams do we have available with the list?

**Comment by BellezaEmporium** [comment, marked as answer]
https://github.com/iptv-org/iptv/blob/master/FAQ.md#why-is-the-channel-on-the-iptv-orggithubio-but-not-in-the-playlist

**Reply by kratoz29** [reply]
Oh damn, my bad, I really missed this in the FAQ, well that pretty much solves my doubt :)
